playground/LoadTest/Registrations.cs

```diff
@@ -470,7 +470,12 @@ private static void RegisterRestObjects(IRegistrator container)
                 container.Register<IModelConverter<WorkPriceEx>, WorkHourPriceModelConverter>(Reuse.Scoped, ifAlreadyRegistered: IfAlreadyRegistered.Replace);
                 container.Register<IModelConverter<TemporaryProjectTravelExpenseModel>, TemporaryProjectTravelsModelConverter>(Reuse.Scoped, ifAlreadyRegistered: IfAlreadyRegistered.Replace);
                 container.Register<PsaReportFactory, PsaReportFactory>(Reuse.Singleton, ifAlreadyRegistered: IfAlreadyRegistered.Throw);
-                container.RegisterDelegate<ReportControllerFactory<IPsaContext>>(c => PsaReportFactory.Factory, Reuse.Singleton, ifAlreadyRegistered: IfAlreadyRegistered.Throw);
+                container.RegisterDelegate<ReportControllerFactory<IPsaContext>>(
+                    c =>
+                    {
+                        return PsaReportFactory.Factory;
+                    }, Reuse.Singleton, ifAlreadyRegistered: IfAlreadyRegistered.Throw
```

Comment:
Its Delegate registration but has Reuse.Singleton so it should be called only once?